common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -109,7 +109,8 @@ static Level parseLevel(String levelStr) {
         Level level = Level.parseLevel(levelStr);
 
         MAX_RECORDS = SystemPropertyUtil.getInt(PROP_MAX_RECORDS, DEFAULT_MAX_RECORDS);
-
+        long maxRecordsSampled = MAX_RECORDS * 10L;
```

Comment:
@Scottmitch do you think we should add another property to change the `10L` here ? I guess it is fine for now and we can just add it later on if needed tho.